### PR TITLE
Force stop consensus

### PR DESCRIPTION
**Pull request by AurelienFT**
Fix https://github.com/massalabs/massa/issues/3766

### CODE REVIEW

**Review by aoudiamoncef — CHANGES_REQUESTED** [review]

File: massa-consensus-worker/src/manager.rs [hunk 1]
```diff
@@ -13,6 +13,7 @@ impl ConsensusManager for ConsensusManagerImpl {
         info!("stopping consensus worker...");
         // join the consensus thread
         if let Some((tx, join_handle)) = self.consensus_thread.take() {
+            tx.send(ConsensusCommand::Stop).unwrap();
```
[the remark this answers is not in the document]
Comment (modship) [reply massa-consensus-worker/src/manager.rs:16]: safe to unwrap here ?

[the remark this answers is not in the document]
Comment (AurelienFT) [reply massa-consensus-worker/src/manager.rs:16]: It's a stop function we always use except in those. I will change to an except.

File: massa-consensus-worker/src/worker/main_loop.rs [hunk 2]
```diff
@@ -59,12 +63,14 @@ impl ConsensusWorker {
     fn wait_slot_or_command(&mut self, deadline: Instant) -> WaitingStatus {
         match self.command_receiver.recv_deadline(deadline) {
             // message received => manage it
-            Ok(command) => {
-                if let Err(err) = self.manage_command(command) {
+            Ok(command) => match self.manage_command(command) {
+                Ok(true) => WaitingStatus::Interrupted,
```
[the remark this answers is not in the document]
Comment (AurelienFT) [reply massa-consensus-worker/src/worker/main_loop.rs:67]: Making an enum called `ShouldStop` with variant `Yes` and `No` ? That's the point of a bool. I don't really understand there

File: massa-consensus-worker/src/manager.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (AurelienFT) [reply massa-consensus-worker/src/manager.rs]: Done

**Review by aoudiamoncef — APPROVED** [review]

**Pull request merged**